devices/lidl.js

```diff
@@ -529,12 +711,152 @@ module.exports = [
         model: 'PSBZS A1',
         vendor: 'Lidl',
         description: 'Parkside smart watering timer',
-        fromZigbee: [fz.ignore_basic_report],
-        toZigbee: [tz.on_off, tz.lidl_watering_timer],
-        onEvent: tuya.onEventSetTime,
-        configure: async (device, coordinatorEndpoint, logger) => {},
-        exposes: [e.switch(), exposes.numeric('timer', ea.SET).withValueMin(1).withValueMax(10000)
-            .withUnit('min').withDescription('Auto off after specific time.')],
+        fromZigbee: [fz.ignore_basic_report, fz.ignore_tuya_set_time, fz.ignore_onoff_report, tuya.fz.datapoints],
+        toZigbee: [
+            {
+                ...tuya.tz.datapoints,
+                key: ['state', 'timer', 'reset_frost_lock', 'schedule_periodic', 'schedule_weekday',
```

Comment:
Please add these keys directly to `tuya.tz.datapoints`, this line should just be `toZigbee: [tuya.tz.datapoints]`